inc/features/blocks/namespace.php

```diff
@@ -190,21 +205,49 @@ function handle_views_request( WP_REST_Request $request ) : WP_REST_Response {
 }
 
 /**
- * Map aggregate bucket data to response format.
+ * Map event type aggregate bucket data to response format.
  *
- * @param array $bucket Elasticsearch aggregate bucket data.
+ * @param array $event_buckets Elasticsearch aggregate buckets data.
  * @return array
  */
-function map_terms_bucket( array $bucket ) : array {
```

Comment:
Is this function completely removed? Or just moved down the file? If it's removed, do we need it?

Reply:
I just renamed it, it's no longer needed